private-templates-service/adaptivecards-templating-service/src/storageproviders/MongoDBProvider.ts

```diff
@@ -28,7 +28,7 @@ export class MongoDBProvider implements StorageProvider {
     if (query.tags && query.tags.length) {
       templateQuery.tags = {
         $all: clone(query.tags).map(x => {
-          return x.toLocaleLowerCase();
+          return new RegExp(`^${x}$`, "i");
```

Comment:
This change will be added with https://github.com/microsoft/adaptivecards-templates/pull/208. I've added it here for correct functioning of All Cards in case this PR gets merged first.